performance/Bakery-Boulangerie/APABakery.tla

```diff
@@ -470,6 +548,7 @@ ISpec == IInit /\ [][Next]_vars
              
 =============================================================================
 \* Modification History
+\* Last modified Wed Aug 14 13:37:20 CEST 2019 by tthai
```

Comment:
This line doesn't seem right

Reply:
I think I copied it over from the OOPSLA artifact